src/components/Register/CreateMenu/CreateMenu.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react';
+import MenuForm from '../../Forms/MenuForm/MenuForm';
+import axios from 'axios';
+import {Redirect} from 'react-router-dom';
 
 const CreateMenu = props => {
 
+	const [menu, updateMenu] = useState({})
+	const [redirect, activateRedirect] = useState(false)
+
+	const submitMenu = async () => {
+		if(Object.keys(menu).length>1){
+			console.log('submitting menu')
+			try{
+				await axios.post('/api/add-menus', {menu})
+			}catch (error){
+				console.log(error)
+			}
+			activateRedirect(true)
+		} else {
+			console.log('first render')
+		}
+	}
+	//find a way to prevent useEffect from firing when component mounts
+	useEffect(()=>{submitMenu()},[menu])
+	
 	return (
-		<></>
+		<>
+			{redirect?<Redirect to='/home'/>:<></>}
```

Comment:
logical and probably fits better here

Reply:
Wouldnt you want that top empty carrot on 28 not to be there at all?